tests/test_log_out.py

```diff
@@ -0,0 +1,16 @@
+from .pages.formLogin import LogIn
+from .pages.logout import Logout
+
+
+def test_log_in_success(driver):
+    form = LogIn(driver)
```

Comment:
`    form = LogIn(driver)
    form.navigate_to_form()
    form.enter_login_username("student")
    form.enter_login_password("Password123")
    form.click_login_button()`
    
above lines related more to Arrange(https://docs.pytest.org/en/7.1.x/explanation/anatomy.html?highlight=arrange%20act%20assert) better to put it in fixture

Reply:
ok, it is done